app/views/root/_publication.html.erb

```diff
@@ -12,7 +12,7 @@
 
     <% if publication.published? %>
       <%= link_to "#{Plek.new.website_root}/#{publication.slug}", class: 'link-muted' %>
-    <% elsif publication.safe_to_preview? %>
+    <% elsif !publication.archived? %>
```

Comment:
Oh, thanks. I only looked at the Edition `safe_to_preview?`, I didn't realise GuideEditions were special. In this case, to avoid getting too deep into content-models, I'll just do the former: `&& !publication.archived?`.